wallet-new/src/Cardano/Wallet/API/V1/Errors.hs

```diff
@@ -50,6 +50,7 @@ data WalletError =
     | SomeOtherError { weFoo :: !Text, weBar :: !Int }
     | MigrationFailed { weDescription :: !Text }
     | WalletNotFound
+    | NoPassPhrase
```

Comment:
I think this shouldn't be an error. A user can surely not pass a spending password, as it's optional. Maybe let's rename it to `InvalidSpendingPassword`, to be thrown when the passwords do not match?